tests/Unit/CommandLine/Library/GitChangeSetFilterTest.php

```diff
@@ -47,11 +47,11 @@ protected function tearDown(): void
      */
     public function filterByBlacklistAndWhitelistWithoutFilter()
     {
-        $whitelistedDirectory = $this->blacklistedDirectory . '/wumpe';
+        $whitelistedDirectory = $this->blacklistedDirectory . 'wumpe';
         $whitelistedFile = $whitelistedDirectory . '/binNochDa';
         $mockedFileList = new GitChangeSet(
             [
-                $this->blacklistedDirectory . '/sowas',
+                $this->blacklistedDirectory . 'sowas',
```

Comment:
verstehe den umbau der verzeichnispfade auf trailing slashes in der blacklist factory nicht so ganz, für mich sieht das komplizierter aus als vorher. GitChangeSet arbeitet ohne trailing slash, blacklist mit. sollen gleichnamige verzeichnisse und dateien abgrenzbar sein? gibt's auf normalen filesystemen doch gar nicht? was ist der bug, der damit gefixt wird? hilfe!

Reply:
Es ist auch maximal doof und ich bin so kurz davor das alles noch mal neu zu implementieren. Eigentlich sollten jetzt alle Stellen, die mit blacklisting zu tun haben auf trailing slash umgestellt sein. Das muss so weil dass die Tools brauchen. Die anderen Verzeichnisse werden ohne trailing slash behandelt. Das muss auch so weil das die Tools brauchen 🙄  

Ich habe leider das "Templating" für die Terminalkommands an einer Stelle zusammengefasst. Jetzt bedingen sich die Tools gegenseitig und das ist nicht nur doof sonder fällt mir jetzt hier auf die Füße.

Reply:
die tools sind schuld ist erstmal ein valider grund ;)